generators/server/templates/src/test/resources/config/application.yml.ejs

```diff
@@ -64,6 +64,9 @@ spring:
   profiles:
     # Uncomment the following line to enable tests against production database type rather than H2, using Testcontainers
     #active: testcontainers
+<%_ } else if (databaseTypeCouchbase) { _%>
+  profiles:
+    active: test
```

Comment:
```suggestion
```

If possible I would prefer adding
```
@ActiveProfiles(JHipsterConstants.SPRING_PROFILE_TEST)
```
To [IntegrationTest](https://github.com/jhipster/generator-jhipster/blob/ba83d069251ec488e93fceb2379ddf49be04ae7e/generators/server/templates/src/test/java/package/IntegrationTest.java.ejs#L61).

I don't think it needs be specific to couchbase.